.github/workflows/build-test.yml

```diff
@@ -194,13 +192,15 @@ jobs:
           cargo fmt -- --files-with-diff --check
       - name: clippy
         run: |
+          rustup update
           cd fuzz
           cargo clippy
       - name: cargo-fuzz
         run: cargo +nightly install cargo-fuzz
       # TODO: it would be nice to save and restore the corpus between runs
       - name: build corpus
         run: |
+          cargo update -p proc-macro2  # Temporary fix to get the nightly build to run correctly
```

Comment:
we fixed this by updating the `Cargo.lock` file in the repository. I think I would prefer it that way

Reply:
The cargo.lock file has been updated by running cargo update for proc-macro2